test/MVCFramework45.FunctionalTests/project.json

```diff
@@ -14,11 +14,12 @@
         "wwwroot",
         "Views"
       ]
-    }
+    },
+    "nowarn": [ "CS0618" ]
```

Comment:
We are using tons of obsolete API's from base sdk, this is just to clean up build output from things we are ignoring anyway.